qctrlopencontrols/driven_controls/driven_control.py

```diff
@@ -278,25 +278,21 @@ def directions(self):
         amplitudes = np.sqrt(
             self.amplitude_x ** 2 + self.amplitude_y ** 2 + self.detunings ** 2
         )
+
+        amplitudes = np.where(np.isclose(amplitudes, 0.0), 1.0, amplitudes)
```

Comment:
Is there a test for this class? If so, might be good just to add a small test for this case, to prevent it breaking again in the future.

Reply:
Sure. There was already one test for the class, but it seemed kind of bloated already, so I added another short test for the directions method